Source/Core/Core/IOS/Network/KD/NWC24Config.cpp

```diff
@@ -228,4 +228,16 @@ std::string NWC24Config::GetCheckURL() const
   const size_t size = strnlen(m_data.http_urls[1], MAX_URL_LENGTH);
   return {m_data.http_urls[1], size};
 }
+
+std::string NWC24Config::GetSendURL() const
```

Comment:
Is there a reason you return a `std::string` here but a `std::string_view` in `GetPassword()` below...?

Reply:
the HTTP functions take a `const std::string&`.